packages/edit-post/src/components/header/plugin-more-menu-item/test/__snapshots__/index.js.snap

```diff
@@ -23,7 +23,7 @@ exports[`PluginMoreMenuItem renders menu item as button properly 1`] = `
     >
       <svg
         aria-hidden="true"
-        className="dashicon dashicons-smiley"
+        className="dashicon dashicons-smiley components-icon-button components-menu-item__button has-icon has-text"
```

Comment:
This is a bug which needs to be fixed. All those class names shouldn't be passed down. They are already applied to the button.